transient/transient.py

```diff
@@ -374,12 +373,21 @@ def run(self) -> None:
             # The user almost certainly doesn't intend to pass a relative path,
             # so make it absolute
             absolute_local_path = os.path.abspath(local)
+
+            # Our preexec_fn kills the server early if it's launched from the thread.
+            sftp_proc = subprocess.Popen(
```

Comment:
Given that this is an implementation detail of the folder mounting, I'm not sure it should be spawned here.

Reply:
Fixed by leaving SSHFS threads running.